Packs/CortexXDR/Playbooks/Cortex_XDR_Alerts_Handling_README.md

```diff
@@ -49,7 +49,7 @@ This playbook does not use any scripts.
 | PaloAltoNetworksXDR.Incident.alerts.actor_process_signature_status | Signature status "Signed" "Invalid Signature" "Unsigned" "Revoked" "Signature Fail" "N/A" "Weak Hash" | unknown |
 | PaloAltoNetworksXDR.Incident.alerts.actor_process_signature_vendor | Singature vendor name | unknown |
 | PaloAltoNetworksXDR.Incident.alerts.action_process_image_sha256 | Image SHA256 | unknown |
-| PaloAltoNetworksXDR.Incident.alerts.is_whitelisted | Is on allow list "Yes" "No" | unknown |
+| PaloAltoNetworksXDR.Incident.alerts.is_whitelisted | Is whitelisted "Yes" "No" | unknown |
```

Comment:
Please change from whitelist to allow list. 
We dont want to use those work (white / black list :) )

Reply:
done